### ISSUE TITLE
Quiet Test Cases

### ISSUE THREAD

**Issue by mgaudet** [eclipse/openj9#200]
There's a number of tests in `tests` that put out quite a bit of test information even when succeeding, leading to a difficult to parse volume of data. 

This epic item should track noisy tests, and subsequent issues can be opened to actually quiet them: 

- [ ] `j9vm.test.romclasscreation.ROMRetentionAfterRAMFailureTest` Produces hundreds of lines of output. 
- [x] `j9vm.test.arraycopy.MultiThreadedArrayCopyTest` 

We probably should aim for a way to change logging levels that's reasonably universal tho.

**Comment by smlambert** [comment]
Good idea!  I think we should also take a look at any steps in the build process that produces lots of output (and correcting by setting unzip steps to quiet mode, etc).

**Comment by smlambert** [comment]
Re-opening, as there are 2 test cases mentioned in this epic (we will still need to cleanup ROMRetentionAfterRAMFailureTest)

**Comment by Sundar1709** [comment]
please provide some more details to understand this scenario better

**Comment by adityaRaj369** [comment]
Hi, I’d like to work on quieting `j9vm.test.romclasscreation.ROMRetentionAfterRAMFailureTest` for this issue. Is this still valid and okay for me to take?
